alma/alma.php

```diff
@@ -151,12 +151,9 @@ public function handlePSModule()
      */
     public function checkCompatibilityPSModule()
     {
-        return false;
```

Comment:
I feel like this PR includes changes that are not related to the Cart Item design fix ?

Reply:
@Benjamin-Freoua-Alma  Should this PR be rebased or target a different branch for merging ? 🤔

Reply:
@FranceBe I rebased the main branch of cart-item and I forgot to rebase this too ^^
I fixed it